### PR TITLE
Add buf linter call to Makefile

### PR DESCRIPTION
**Pull request by realloc**
Added `buf` linter call to `Makefile` and made soe fixes to make it silent.

### CODE REVIEW

File: accounting/service.proto [hunk 1]
```diff
@@ -1,17 +1,17 @@
 syntax = "proto3";
 
-package accounting;
+package neo.fs.accounting.v2;
```
Comment (alexvanin) [inline accounting/service.proto:3]: Don't you want to use `neo.fs.v2.package-name`? This will match `neofs-api-go` import structure.

Comment (realloc) [reply accounting/service.proto]: typo. fixed.

File: object/types.proto [hunk 2]
```diff
@@ -12,34 +12,46 @@ import "service/verify.proto";
...
-  REGULAR = 0;
+  OBJECT_TYPE_UNSPECIFIED = 0;
+
   // Used internally to identify deleted objects
-  TOMBSTONE = 1;
+  OBJECT_TYPE_TOMBSTONE = 1;
+
   // Identifies that the object holds StorageGroup information
-  STORAGE_GROUP = 2;
+  OBJECT_TYPE_STORAGE_GROUP = 2;
```
Comment (alexvanin) [inline object/types.proto:21]: Autogenerated code have enum prefix in these constant. `ObjectType_OBJECT_TYPE_STORAGE_GROUP` will look ugly.

File: refs/types.proto [hunk 3]
```diff
@@ -1,9 +1,9 @@
 syntax = "proto3";
 
-package refs;
+package neo.fs.v2.refs;
 
-option go_package = "github.com/nspcc-dev/neofs-api-go/refs";
-option csharp_namespace = "NeoFS.API.Refs";
+option go_package = "github.com/nspcc-dev/neofs-api-go/v2/refs";
```
Comment (alexvanin) [inline refs/types.proto:5]: `option go_package = "github.com/nspcc-dev/neofs-api-go/v2/refs;refs"` ?

File: service/meta.proto [hunk 4]
```diff
@@ -28,67 +31,87 @@ message Version {
...
-      OBJECT_DELETE = 4;
+      VERB_DELETE = 5;
+
       // Refers to object.GetRange RPC call
-      OBJECT_RANGE = 5;
+      VERB_RANGE = 6;
+
       // Refers to object.GetRangeHash RPC call
-      OBJECT_RANGEHASH = 6;
+      VERB_RANGEHASH = 7;
```
Comment (alexvanin) [inline service/meta.proto:76]: `OBJECT` prefix considered that this enum can be expanded for non object related tokens, therefore non object related verbs. I think it was good idea.

File: storagegroup/types.proto [hunk 5]
```diff
@@ -1,9 +1,9 @@
 syntax = "proto3";
 
-package storagegroup;
+package neo.fs.v2.storagegroup;
 
-option go_package = "github.com/nspcc-dev/neofs-api-go/storagegroup";
-option csharp_namespace = "NeoFS.API.StorageGroup";
+option go_package = "github.com/nspcc-dev/neofs-api-go/v2/storagegroup";
```
Comment (alexvanin) [inline storagegroup/types.proto:5]: `option go_package = "github.com/nspcc-dev/neofs-api-go/v2/storagegroup;storagegroup"`?

Comment (realloc) [reply storagegroup/types.proto:5]: typo. fixed.

File: refs/types.proto (hunk 3 above)
Comment (realloc) [reply refs/types.proto:5]: typo.fixed.

File: object/types.proto (hunk 2 above)
Comment (realloc) [reply object/types.proto:21]: Agree

File: service/meta.proto (hunk 4 above)
Comment (realloc) [reply service/meta.proto:76]: I'd propose to separate contexts by type, i.e. have different context structures for each service type. Maybe we should keep verb enum as is for now and change it in the following PRs.

File: object/types.proto (hunk 2 above)
Comment (realloc) [reply object/types.proto]: Agree. Changed for all enums except default zero-valued first UNSPECIFIED fields. They must be unique package-wide.

**Pull request merged**